vta/tutorials/frontend/deploy_vision_on_vta.py

```diff
@@ -100,7 +115,7 @@
     # the host, make sure you've set the variables below to the IP of
     # your board.
     device_host = os.environ.get("VTA_PYNQ_RPC_HOST", "192.168.2.99")
-    device_port = os.environ.get("VTA_PYNQ_RPC_PORT", "9091")
+    device_port = int(os.environ.get("VTA_PYNQ_RPC_PORT", "9091"))
     if not tracker_host or not tracker_port:
         remote = rpc.connect(device_host, int(device_port))
```

Comment:
why convert `device_port` to int again?

Reply:
that's a carry over from old code and should be cleaned up, thanks for the catch!